src/Sylius/Behat/Resources/config/services/api.xml

```diff
@@ -97,6 +103,12 @@
             <argument>shipping-methods</argument>
         </service>
 
+        <service id="sylius.behat.api_platform_client.shop_billing_data" class="Sylius\Behat\Client\ApiPlatformClient">
+            <argument type="service" id="test.client" />
+            <argument type="service" id="sylius.behat.shared_storage" />
+            <argument>shop-billing-datas</argument>
```

Comment:
I'm wondering if it shouldn't be `/channels/1/shop-billing-data` instead of `/channels/1/shop-billing-datas/1`

Reply:
But I don't know how to achieve this :(

Reply:
I agree that it looks better, but I would be for leaving it for refactoring in another PR (if it will be possible of course)